src/stores.ts

```diff
@@ -55,3 +60,10 @@ window.onpopstate = function (event) {
     return p;
   });
 };
+
+export const mapData = writable([]);
```

Comment:
Please add some comments explaining what these global variables are, and which processes update them under what conditions.

Reply:
done.